teams/iaas/onetimes.yml

```diff
@@ -53,3 +53,16 @@ events:
       Dial-In: +49-221-292772-611
       Coordinator: Patrick Thiem <patrick.thiem[at]cloudandheat.com>
     location: "https://conf.scs.koeln:8443/SCS-Tech"
+- summary: "IaaS Breakout room - DNS Standardization"
+    begin: 2024-05-10 14:05:00
```

Comment:
is this really ment to happen in may?

Reply:
thanks for catching that! of course not, this was a c&p mistake :grin: